src/test/apisix/templates/deployment.yaml

```diff
@@ -80,8 +80,8 @@ spec:
       initContainers:
       - name: init-apisix
```

Comment:
better to use `wait-etcd`. `init-apisix` often means populate some data for apisix.

Reply:
done